sdp/processors/langs/armenian.py

```diff
@@ -0,0 +1,90 @@
+import os
+import pandas as pd
+from pathlib import Path
+from sdp.processors.base_processor import BaseProcessor, BaseParallelProcessor, DataEntry
+from sdp.utils.common import load_manifest
+
+
+class GetSource(BaseParallelProcessor):
+    """
+    Processor for extracting source information from file paths and updating the manifest.
+
+    Args:
+    - source_filepath (str): The field containing the file path in the manifest.
```

Comment:
Please remove these dashes from all docstrings. I think Sphinx requires no dashes to render docs properly?

Reply:
done, sorry, forgot about this file